api-conta-bancaria/src/main/resources/db/migration/V001__criando-tabela-cliente.sql

```diff
@@ -0,0 +1,6 @@
+create table cliente(
+id BigInt not null auto_increment,
```

Comment:
gostei.
Um int ja resolvia, mas não tem problema
o primary key por default é not null, nem precisaria informar isso ex: id int auto_increment
Fica de aprendizado para os proximos